versions/windows-server-2022.yml

```diff
@@ -2,6 +2,6 @@ osDistro: windows-server
 osVersion: 2022
 packages:
   - arch: win64
-    ami: ami-094aa6728b151e05a
+    ami: ami-0c808db6baea2d0ed
   - arch: win32
-    ami: ami-094aa6728b151e05a
+    ami: ami-0c25be66a8d4215e5
```

Comment:
Why so many AMIs needed to be updated? Did they get updated recently?

Reply:
Existing AMI's had no images , so thats why updated them.